src/createSynaptik.tsx

```diff
@@ -66,5 +155,6 @@ export function createSynaptik<C extends MapClassInstances<C>>(synapse: Synapse<
     Provider,
     useSynapse,
     connect,
+    ...synapse,
```

Comment:
Why pass this down?

Reply:
For tests and in case we want handles on `updateState` or `getState` in app code - redux exposes these as well.